blocks/README.md

```diff
@@ -94,8 +94,9 @@ wp.blocks.registerBlock( 'myplugin/random-image', {
 		return el( 'form', { onSubmit: setCategory }, children );
 	},
 
-	save: function( attributes ) {
-		return RandomImage( { category: attributes.category } );
+	serialize: function( attributes ) {
+		const src = 'http://lorempixel.com/400/200/' + props.category;
+		return `<img src="${ src }" alt="${ props.category }"/>`;
```

Comment:
I can't help but this documentation only goes to highlight why this is an idea we should avoid:

- What if `category` has a `"` in it?
- Why can't we re-use `RandomImage` to generate the same markup?
- What if my markup is more than just a single HTML element, like a gallery with potentially hundreds of elements?
- What benefit does this ever have that even if your answer to any of the above is "just call `renderToString`", I as the developer must even come to know that `renderToString` is a thing?